ui/webui/test/README.rst

```diff
@@ -92,6 +92,14 @@ to test, you can run tests by picking a program and just executing it against th
 
     test/check-basic --machine=127.0.0.2:22000 --browser 127.0.0.2:9091
 
+Pixel tests
+-----------
+
+The verify test suite contains ["pixel tests"](https://cockpit-project.org/blog/pixel-testing.html).
+Make sure to create the test/reference submodule before running tests which contain pixel tests.::
```

Comment:
I wonder if we can't make the makefile target or the script that runs the tests do that ? Otherwise this might become yet another "why is the thing not working ?" stumbling stone for new and occasional contributors.

Possibly as a follow-up PR. :)

Reply:
I will need to repush here anyway, to bump the testlib dependency, so I will make sure to adjust the makefile as well!

Reply:
Thanks! :)